src/app/functions/server/onerep.ts

```diff
@@ -336,18 +336,13 @@ export async function isEligibleForFreeScan(
 }
 
 export function isEligibleForPremium(
-  user: Session["user"],
```

Comment:
This argument wasn't really used anymore in this function - it was there so we could fetch the user's enabled feature flags, but in a previous PR I changed this function to accept the enabled flags as an argument, and thus the user data was no longer relevant to this function.